### PR TITLE
Fix podman inspect NetworkSettings.Ports

### PR DESCRIPTION
**Pull request by tossmilestone**
/assign @aojea

### CODE REVIEW

File: pkg/cluster/internal/providers/podman/provider.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func (p *provider) GetAPIServerEndpoint(cluster string) (string, error) {
 		cmd := exec.Command(
 			"podman", "inspect",
 			"--format",
-			"{{ json .NetworkSettings.Ports }}",
+			"json {{ .NetworkSettings.Ports }}",
```
Comment (aojea) [inline pkg/cluster/internal/providers/podman/provider.go:199]: previous format was correct, we want to get the subsection in json format, this is the same syntax that podman uses for testing it 
https://github.com/containers/podman/blob/5c6b5ef34905f40562b518799c35be8d06694e65/test/e2e/inspect_test.go#L440
It´s also how docker documents it https://docs.docker.com/engine/reference/commandline/inspect/#get-a-subsection-in-json-format

Comment (tossmilestone) [reply pkg/cluster/internal/providers/podman/provider.go:199]: I got error when execute the command:
```
[root@localhost ~]# podman inspect --format="{{ json .NetworkSettings.Ports }}" kind-control-plane
ERRO[0000] Error printing inspect output: template: all inspect:1: function "json" not defined
```
Is it a not supported podman version?

Comment (aojea) [reply pkg/cluster/internal/providers/podman/provider.go:199]: I've already fixed it in kind
https://github.com/kubernetes-sigs/kind/blob/e36bf401e3d9792956f216855712606b3069d41e/pkg/cluster/internal/providers/podman/provider.go#L254

And podman has fixed it upstream recently.

You have to use kind from master

Comment (tossmilestone) [reply pkg/cluster/internal/providers/podman/provider.go:199]: Thanks @aojea

**Pull request closed without merging**